### PR TITLE
Hybrid SPA sample update 

### PR DESCRIPTION
**Pull request by salman90**
## Purpose

Updated hybrid SPA  to the following:

1. Updated Config file to support Optional claims and hybrid SPA URL
2. Updated sample.json to include Optional claims and hybrid SPA URL
3. Updated readme to include to topology image 

Thanks for the feedback!

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial.git
cd  .\ms-identity-javascript-react-tutorial\6-AdvancedScenarios\4-sign-in-hybrid\App
git checkout update-hybridSpa-sample
npm install
npm start
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by derisen — APPROVED** [review]
LGTM, thanks @salman90

File: 6-AdvancedScenarios/4-sign-in-hybrid/AppCreationScripts/sample.json [hunk 1]
```diff
@@ -1,118 +1,116 @@
...
+        "Author": "Salman Salem",
+        "Title": "Hybrid SPA sample",
+        "Level": 300,
+        "Client": "React SPA",
+        "Service": "Node.js webapp",
+        "RepositoryUrl": "ms-identity-javascript-react-tutorial",
+        "Endpoint": "AAD v2.0",
+        "Languages": ["JavaScript"],
+        "Description": "This sample demonstrates an Express web app calling a React SPA using the Hybrid SPA flow",
+        "Products": ["React", "Express"]
```
Comment (kalyankrishna1) [inline 6-AdvancedScenarios/4-sign-in-hybrid/AppCreationScripts/sample.json:12]: >React [](http://example.com/codeflow?start=22&length=5)

add msal and "azure active directory"

File: 6-AdvancedScenarios/4-sign-in-hybrid/README.md [hunk 2]
```diff
@@ -123,12 +125,12 @@ There is one project in this sample. To register it, you can:
...
 1. Select **Register** to create the application.
 1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
-1. In the app's registration screen, select **Authentication** in the menu.
+1. In the app's registration screen, select **Authentication** blade.
    * If you don't have a platform added, select **Add a platform** and select the **Web** option.
    * In the **Redirect URIs** section, enter the following redirect URIs.
      * `http://localhost:5000/redirect`
    * Click on the **Add a platform** button in the **Platform configurations** section of the page.
-     * Select the **Single-page application** button and enter `https://localhost:5000/blank` as the **Redirect URI** and click the **Configure** button.
+     * Select the **Single-page application** button and enter `http://localhost:5000/blank` as the **Redirect URI** and click the **Configure** button.
```
Comment (kalyankrishna1) [inline 6-AdvancedScenarios/4-sign-in-hybrid/README.md:133]: >blank [](http://example.com/codeflow?start=86&length=5)

really, "blank"

**Comment by kalyankrishna1** [comment]
update topology for two calls to Graph and remove one AAD icon

**Review by kalyankrishna1 — APPROVED** [review]
:shipit:

**Pull request merged**